tests/testthat/test_merge.R

```diff
@@ -70,17 +84,92 @@ test_that("assign global identifiers", {
   
   expect_equal(unique(out$lookup$VPU), c("01", "02"))
   
-  in1 <- hydrofab::read_hydrofabric(agg_gpkg_1, verbose = FALSE)
-  in2 <- hydrofab::read_hydrofabric(agg_gpkg_2, verbose = FALSE)
+  unlink(refac_gpkg_1)
+  unlink(refac_gpkg_2)
+})
+
+test_that("assign global identifiers aggregated", {
+  
+  agg_gpkg_1 <- tempfile(fileext = "_01.gpkg")
+  file.copy(new_hope_agg, agg_gpkg_1)
+  
+  agg_gpkg_2 <- tempfile(fileext = "_02.gpkg")
+  file.copy(new_hope_agg, agg_gpkg_2)
+  
+  in1 <- read_hydrofabric(agg_gpkg_1, verbose = FALSE)
+  in2 <- read_hydrofabric(agg_gpkg_2, verbose = FALSE)
   
   out <- assign_global_identifiers(c(agg_gpkg_1, agg_gpkg_2), 
                                    flowpath_layer = "flowpath", 
                                    divide_layer = "divides", 
                                    mapped_POI_layer = "mapped_POIs", 
                                    lookup_table_layer = "lookup_table", 
-                                   catchment_network_layer = "catchment_network", 
+                                   flowpath_edge_list = "catchment_network", 
                                    overwrite = TRUE, verbose = FALSE)
   
   expect_equal(seq((nrow(in1$flowpaths) + nrow(in2$flowpaths))), out$lookup$newID)
   
+  agg_gpkg_1 <- tempfile(fileext = "_14.gpkg")
+  agg_gpkg_2 <- tempfile(fileext = "_15.gpkg")
+  
+  file.copy(new_hope_agg, agg_gpkg_1)
+  file.copy(new_hope_agg, agg_gpkg_2)
+  
+  lu <- sf::read_sf(agg_gpkg_1, "lookup_table")
+  
+  # These were chosen by hand 
```

Comment:
This is where things get wild. I'm mocking a VPU outlet and just making sure the code does what we expect to the mock data. The indirection of having to use the comid to discern the mapping is a PIA, but it functions and the test should be fairly robust.